pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/StudyBuilder.java

```diff
@@ -677,6 +681,47 @@ private void insertStatistics(Handle handle, long studyId) {
         }
     }
 
+
+    private void insertCategories(Handle handle, long studyId) {
+        if (!cfg.hasPath("categories")) {
+            return;
+        }
+        ActivityCategoryDao categoryDao = handle.attach(ActivityCategoryDao.class);
+        for (Config categoryEntry : cfg.getConfigList("categories")) {
+            String categoryName = categoryEntry.getString("categoryName");
+            String categoryCode = categoryEntry.getString("categoryCode");
+            List<FormGroupDef> subForms =
+                    categoryEntry.getConfigList("subForms").stream().map(this::getSubForm).collect(Collectors.toList());
```

Comment:
References to use of the word subforms is a little confusing to me as we generally don't refer to activities as forms (though I think maybe we should have called them forms! But it is what it is).
Maybe just call it activities?